docs/index.md

```diff
@@ -124,7 +124,7 @@ td.save(feature_join)
 
 A short introduction to the Scala API:
 ```scala
-import com.logicalclocks.hsfs.spark._
+import com.logicalclocks.hsfs._
```

Comment:
Why are you doing this, this was correct.

Reply:
Not sure why this happended, reverted it